parsers/AR.py

```diff
@@ -37,9 +55,15 @@ def fetch_production(
     non_renewables_production: Dict[str, dict] = non_renewables_production_mix(
         zone_key, current_session
     )
-    renewables_production: Dict[str, dict] = renewables_production_mix(
-        zone_key, current_session
-    )
+
+    if zone_key == "AR":
+        renewables_production: Dict[str, dict] = renewables_production_mix(
+            zone_key, current_session
+        )
+    else:
+        renewables_production: Dict[str, dict] = regional_renewables_production_mix(
```

Comment:
```suggestion
        renewables_production = regional_renewables_production_mix(
```
Since you added a typed return the type can be inferred from the function, so there's no need to put the type again.